### DISCUSSION TITLE
What will happen with V3 apps on 1st Jan 2025?

### DISCUSSION THREAD

**Discussion by oskarpearson** [serverless/serverless#12899] [Q&A] [answered]
Hi All

Are there any details available about what will happen with V3 apps on 1st Jan, 2025? Specifically, will deploys and apps using the V3 framework continue to work as they do before that date?

https://www.serverless.com/framework/docs/guides/upgrading-v4#license-changes explains that the V3 framework "will continue to be maintained via critical security and bug fixes through 2024".

I have a few very-low-criticality apps that I will need to upgrade to V4, but would prefer to do them sometime in January, since there are more pressing things going on at the moment.

Is there any indication that V3 apps will fail or fail to deploy from the 1st? If the devs are planning on removing the v3 framework repositories, for example, it may break deploys.

Thanks!

**Comment by garethmcc** [comment, marked as answer]
Hi there @oskarpearson  and for anyone else reading this. Nothing will change with V3 after 2024. It will still be available on NPM and on GitHub. All already deployed services will continue working as before and you can still continue to use V3 as you have, there just will be no updates for bug fixes, security issues or new features in 2025 and onwards.
